runtime_test.go

```diff
@@ -1,72 +1,72 @@
 package main
 
 import (
-  "os"
-  "github.com/awslabs/goformation"
-  "github.com/awslabs/goformation/resources"
-  . "github.com/onsi/ginkgo"
-  . "github.com/onsi/gomega"
+	"github.com/awslabs/goformation"
+	"github.com/awslabs/goformation/resources"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
 )
 
 var _ = Describe("sam", func() {
-  Describe("runtime", func() {
+	Describe("runtime", func() {
```

Comment:
I don't want to start another war here, but personally I don't like 8 char tabs that `go fmt` uses. We can use another linter tool if needed, but not `go fmt`